Ghidra/Features/Base/src/main/java/ghidra/app/util/opinion/OmfLoader.java

```diff
@@ -478,6 +547,11 @@ private void processExternalSymbols(OmfFileHeader header, Program program, TaskM
 		SymbolTable symbolTable = program.getSymbolTable();
 		Language language = program.getLanguage();
 
+		Map<String, OmfSymbol> publicSymbols = header.getPublicSymbols()
+				.stream()
+				.flatMap(symbolRec -> symbolRec.getSymbols().stream())
+				.collect(Collectors.toMap(sym -> sym.getName(), Function.identity()));
```

Comment:
> Task :Base:compileJava
/tmp/ghidra/Ghidra/Features/Base/src/main/java/ghidra/app/util/opinion/OmfLoader.java:553: error: cannot find symbol
                                .collect(Collectors.toMap(sym -> sym.getName(), Function.identity()));
                                                                                        ^
  symbol:   method identity()
  location: interface Function

Reply:
so eclipse was fine but gradle wasn't?  Thats kinda weird.

If you can't figure it out, you could replace it with "val -> val"

Reply:
`import java.util.function.Function;` is the key on my system ...